storage/innobase/log/log0recv.cc

```diff
@@ -4439,7 +4439,7 @@ static dberr_t recv_rename_files()
         err= space->rename(new_name, false);
         if (err != DB_SUCCESS)
           sql_print_error("InnoDB: Cannot replay rename of tablespace "
-                          UINT32PF " to '%s: %s", new_name, ut_strerr(err));
+                          UINT32PF " to '%s': %s", id, new_name, ut_strerr(err));
```

Comment:
`git blame` 28b27b96d0

Reply:
The `'` is a cosmetic fix.

Reply:
> Please file a JIRA and reference it in the backport commit message

I have included this in [MDEV-35431](https://jira.mariadb.org/browse/MDEV-35431).